docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -1,34 +1,57 @@
-==========================
-File system access control
-==========================
+=========================
+File-based access control
+=========================
 
-This access control plugin allows you to specify authorization rules in a JSON file.
+To secure access to data in your cluster, you can implement file-based access
+control where access to data and operations is defined by rules declared in
+manually-configured JSON files.
+
+There are two types of file-based access control:
+
+* **System-level access control** uses the access control plugin with a single
+  JSON file that specifies authorization rules for the whole cluster.
+* **Catalog-level access control** uses individual JSON files for each catalog
+  for granular control over the data in that catalog, including column-level
+  authorization.
+
+File-based access control process from the top to bottom of the JSON file with
+an implicit deny. If a rule matches the user then the corresponding operation
+is allowed. If no rules match, the operation is denied.
+
+.. _system-file-based-access-control:
+
+System-level access control files
+=================================
+
+The access control plugin allows you to specify authorization rules for the
+cluster in a single JSON file.
 
 Configuration
 -------------
 
-To use this plugin, add an ``etc/access-control.properties`` file containing two
-required properties: ``access-control.name``, which must be equal to ``file``, and
-``security.config-file``, which must be equal to the location of the config file.
-For example, if a config file named ``rules.json``
-resides in ``etc``, add an ``etc/access-control.properties`` with the following
+To use the access control plugin, add an ``etc/access-control.properties`` file
+containing two required properties: ``access-control.name``, which must be set
+to ``file``, and ``security.config-file``, which must be set to the location
+of the config file. For example, if a config file named ``rules.json`` resides
+in ``etc``, add an ``etc/access-control.properties`` with the following
 contents:
 
 .. code-block:: text
 
    access-control.name=file
    security.config-file=etc/rules.json
 
-The config file is specified in JSON format. It contains rules that define
-which users have access to which resources. The rules are read from top to bottom
-and the first matching rule is applied. If no rule matches, access is denied.
+The config file is specified in JSON format. It contains rules that define which
+users have access to which resources. The rules are read from top to bottom and
+the first matching rule is applied. If no rule matches, access is denied.
```

Comment:
I'd recommend to keep this paragraph and remove the one at line 17 instead. The one at line 17 is introduced prematurely and it's possible in future catalog rules and system rules will have different semantics/mechanics.

Reply:
Done